src/core_plugins/kibana/public/management/sections/settings/lib/get_aria_name.js

```diff
@@ -0,0 +1,13 @@
+
+import { words } from 'lodash';
+
+/**
+ * @name {string} the name of the configuration object
+ * @returns {string} a space demimited, lowercase string with
+ *          special characters removed.
+ *
+ * Example: 'xPack:fooBar:foo_bar_baz' -> 'x pack foo bar foo bar baz'
```

Comment:
Probably okay since we're not relying on a certain delimiter and I'm guessing/hoping the lodash `words` method is robust enough to handle any future settings